tests/cecil-tests/Helper.cs

```diff
@@ -50,7 +50,7 @@ public static void AssertFailures (Dictionary<string, string> currentFailures, H
 			AssertFailures<string> (currentFailures, knownFailures, nameOfKnownFailureSet, message, (v) => v);
 		}
 
-		public static void AssertFailures<T> (Dictionary<string, T> currentFailures, HashSet<string> knownFailures, string nameOfKnownFailureSet, string message, Func<T, string> failureToString) where T : notnull
+		public static void AssertFailures<T> (Dictionary<string, T> currentFailures, HashSet<string> knownFailures, string nameOfKnownFailureSet, string message, Func<T, string> failureToString) where T : notnull, IComparable
```

Comment:
I have a feeling that the compiler should have been smart enough to let us know that the method has a call of OrderBy that required T to be IComparable. Roslyn, I'm disappointed.

Reply:
man how (un)lucky!